.github/workflows/post-submit.yaml

```diff
@@ -8,31 +8,31 @@ on:
 
 jobs:
   push_to_registry:
-    name: Push a container image to quay.io/medik8s
+    name: Build and push images to quay.io/medik8s
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
       - name: Set up Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
         with:
           go-version: 1.18
 
       - name: Log in to Quay.io
-        uses: docker/login-action@v1
+        uses: docker/login-action@v2
         with:
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
           registry: quay.io
 
       - name: Build and push operator and bundle images, using version 0.0.1, for pushes to main
         if: ${{ github.ref_type != 'tag' }}
-        run: export VERSION=0.0.1 && make docker-build bundle bundle-build docker-push bundle-push
+        run: make container-build container-push
```

Comment:
As far as I see this is not logically equivalent, can you explain the motivation ?

Reply:
> As far as I see this is not logically equivalent,

That's correct, `export VERSION=0.0.1 &&` is not needed after the changes I am suggesting in Makefile

> container-push also runs catalog-build & catalog-push

I replied on the next comment.